VenafiPS/Public/Test-TppToken.ps1

```diff
@@ -0,0 +1,170 @@
+<#
+.SYNOPSIS
+Test if a Tpp token is valid
+
+.DESCRIPTION
+Use the TPP API call 'Authorize/Verify' to test if the current token is valid.
+
+.PARAMETER AuthServer
+Auth server or url, venafi.company.com or https://venafi.company.com.
+If just the server name is provided, https:// will be appended.
+
+.PARAMETER AccessToken
+Access token retrieved outside this module.  Provide a credential object with the access token as the password.
+
+.PARAMETER TppToken
+Token object obtained from New-TppToken
+
+.PARAMETER VenafiSession
+Session object created from New-VenafiSession method.  The value defaults to the script session object $VenafiSession.
+
+.PARAMETER GrantDetail
+Provides detailed info about the token object from the TPP server response as an output.
+PSCustomObject with the following properties:
+    AuthUrl
+    AccessToken
+    RefreshToken
+    Scope
+    Identity
+    TokenType
+    ClientId
+    Expires
+
+.INPUTS
+Accesstoken
+
+.OUTPUTS
+Boolean (default).
```

Comment:
boolean by default, but pscustomobject if `-GrantDetail` provided.  perhaps also mention when it throws an error.

Reply:
Added